source/_layouts/footer.blade.php

```diff
@@ -91,7 +91,7 @@
                 <div class="col-md-8">
                     <ul class="ud-footer-bottom-left">
                         <li>
-                            <a href="javascript:void(0)">{{ $page->t("Privacy policy",current_path_locale($page))}}</a>
+                            <a href="{{ $page->baseUrl }}privacy-policy">{{ $page->t("Privacy policy",current_path_locale($page))}}</a>
```

Comment:
Didn't this way work?
```suggestion
                            <a href="{{ $page->baseUrl }}privacy-policy">{{ $page->t("Privacy policy")}}</a>
```